flocker/node/agents/ebs.py

```diff
@@ -59,7 +59,8 @@
 # Don't use pyOpenSSL in urllib3 - it causes an ``OpenSSL.SSL.Error``
 # exception when we try an API call on an idled persistent connection.
 # See https://github.com/boto/boto3/issues/220
-extract_from_urllib3()
+# New version of botocore fixes this.
+# extract_from_urllib3()
```

Comment:
The issue above is fixed and seems to be in the latest release.
Although without this line we see lots of insecure SSL warnings in the build results....so perhaps I should leave this for now.